pkgs/development/python-modules/mnist/default.nix

```
@@ -0,0 +1,36 @@
+{ buildPythonPackage, fetchFromGitHub, isPy27, lib, mock, numpy, pytest
```

pytestCheckHook will bring pytest into scope
```suggestion
{ buildPythonPackage, fetchFromGitHub, isPy27, lib, mock, numpy